common/lib/xmodule/xmodule/course_module.py

```diff
@@ -303,6 +303,12 @@ class CourseFields(object):
         help=_("Enter the unique identifier for your course's video files provided by edX."),
         scope=Scope.settings
     )
+    facebook_url = String(
```

Comment:
I think we should either generalize or make this Facebook-specific. It's weird to go from specific to general back to specific (in going from Studio to API to app)